xs2a-adapter-service-api-adapter/src/main/java/de/adorsys/xs2a/adapter/http/ApacheUriBuilder.java

```diff
@@ -26,6 +28,20 @@ public UriBuilder queryParam(String name, String value) {
         return this;
     }
 
+    @Override
+    public UriBuilder renameQueryParam(String currentName, String newName) {
+        List<NameValuePair> queryParams = uriBuilder.getQueryParams();
+        uriBuilder.removeQuery();
+        for (NameValuePair queryParam : queryParams) {
+            if (queryParam.getName().equals(currentName)) {
```

Comment:
maybe it's not the case but better to start the comparison with constant/string variable

Reply:
There are variables on both sides.

Reply:
i mean in case if `getName()` is null we get a nullPointer exception. it is not about the current case it is in general

Reply:
O_o